js/screens/SearchResult/SearchResultScreen.js

```diff
@@ -34,45 +38,136 @@ class SearchResultScreen extends Component {
     };
   };
 
+  getChipLabels() {
+    const {
+      laundryTags,
+      occupantTags,
+      otherTags,
+      parkingTags,
+      propertyTags
+    } = this.props;
+
+    // get all the tags and merge them
+    const allTags = Object.assign(
+      JSON.parse(JSON.stringify(laundryTags)),
+      JSON.parse(JSON.stringify(occupantTags)),
+      JSON.parse(JSON.stringify(otherTags)),
+      JSON.parse(JSON.stringify(parkingTags)),
+      JSON.parse(JSON.stringify(propertyTags))
```

Comment:
I would like to know why parsing and stringify props?

Reply:
To create a deep copy of the object.